app/assets/javascripts/app/phone-internationalization.js

```diff
@@ -0,0 +1,57 @@
+import { PhoneFormatter } from 'field-kit';
+
+const I18n = window.LoginGov.I18n;
+const phoneFormatter = new PhoneFormatter();
+
+const PHONE_UNSUPPORTED_AREA_CODES = {
+  648: 'American Samoa',
```

Comment:
Yeah, still working on the direction for how this looks on the server and thinking in that direction.